SimPEG/EM/Analytics/NSEM.py

```diff
@@ -0,0 +1,128 @@
+import numpy as _np
```

Comment:
You can either specify an `__all__` parameter in the file.

https://stackoverflow.com/questions/2187583/whats-the-python-all-module-level-variable-for

Or probably better is to explicitly import things in the higher level module (`Analytics`).

```pyton
from NSEM import AppImpedance_LayeredEarth
from NSEM import hzAnalyticDipoleF
```

It is more verbose. However, it is explicit, and I think a better option than changing expected variable names.